xml/cap_admin_app_domains.xml

```diff
@@ -0,0 +1,145 @@
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+<!-- TODO review if chapter is still required. See:
+  https://github.com/SUSE/doc-cap/issues/283
+  -->
+  <title>Custom Application Domains</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+  In a standard &scf; deployment, applications will use the same domain as the
+  one configured in your <filename>scf-config-values.yaml</filename> for SCF.
+  For example, if your &scf; deployment's domain is set as
+  <replaceable>example.com</replaceable> then an application's domain will be in the
+  form of <literal>myapp.example.com</literal>.
+ </para>
+ <para>
+  This chapter describes the changes required to allow applications to use a
+  separate domain.
+ </para>
+ <sect1 xml:id="sec.cap.customizing-app-domains">
+  <title>Customizing Application Domains</title>
+
+  <para>
+   Begin by adding the following to your
+   <filename>scf-config-values.yaml</filename>. Replace
+   <replaceable>appdomain.com</replaceable> with the domain to use with your applications:
+  </para>
+
+<screen>bosh:
+  instance_groups:
+  - name: api-group
+    jobs:
+    - name: cloud_controller_ng
+      properties:
+        app_domains:
+	- <replaceable>appdomain.com</replaceable>
+</screen>
+
+  <para>
+   If UAA is deployed, pass your UAA secret and certificate to SCF. Otherwise
+   deploy UAA first (See <xref linkend="sec.cap.install-uaa-prod"/>), then
+   proceed with this step:
+  </para>
+
+<screen>&prompt.user;SECRET=$(kubectl get pods --namespace uaa \
+-o jsonpath='{.items[?(.metadata.name=="uaa-0")].spec.containers[?(.name=="uaa")].env[?(.name=="INTERNAL_CA_CERT")].valueFrom.secretKeyRef.name}')
+
+&prompt.user;CA_CERT="$(kubectl get secret $SECRET --namespace uaa \
+-o jsonpath="{.data['internal-ca-cert']}" | base64 --decode -)"
+</screen>
+
+  <para>
+   If this is an initial deployment, use <command>helm install</command> to
+   deploy SCF:
+  </para>
+
+<screen>&prompt.user;helm install suse/cf \
+--name susecf-scf \
+--namespace scf \
+--values scf-config-values.yaml \
+--set "secrets.UAA_CA_CERT=${CA_CERT}"
+</screen>
+
+  <para>
+   If this is an existing deployment, use <command>helm upgrade</command> to
+   apply the change:
+  </para>
+
+<screen>&prompt.user;helm upgrade susecf-scf suse/cf \
+--values scf-config-values.yaml --set "secrets.UAA_CA_CERT=${CA_CERT}"
+</screen>
+
+  <para>
+   Monitor the progess of the deployment:
+  </para>
+
+<screen>&prompt.user;watch -c 'kubectl get pods --namespace scf'</screen>
+
+  <para>
+   When all pods are in a ready state, do the following to confirm custom
+   application domains have been configured correctly.
+  </para>
+
+  <para>
+   Run <command>cf curl /v2/info</command> and verify the SCF domain is not
+   <literal>appdomain</literal>:
+  </para>
```

Comment:
`<replaceable>appdomain</replaceable>`

Reply:
Sorry got ahead of myself and pushed too early